tools/python/influx-migration/influx_migration.py

```diff
@@ -80,6 +87,12 @@ def backup(backup_path, root_token, src_host, bucket_name=None, full=False, skip
         raise ValueError("bucket_name and full not provided, one must be provided")
 
     logging.info("Backing up bucket data and metadata using the InfluxDB CLI")
+    if logging.root.level >= logging.DEBUG:
+        time.sleep(METRICS_SCRAPE_INTERVAL_SECONDS)
+        if full:
+            report_all_bucket_series_count(host=src_host, token=root_token)
```

Comment:
This does not use an org since if a migration is being done without `--csv`, an operator token is required and an operator token will provide access to all buckets in all organizations.